docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -120,107 +120,107 @@ Once the NuGet packages are updated to recent versions, the `<PackageReference>`
+A common source of errors is missing references for APIs that are available for .NET Core but not automatically included in the .NET Core App metapackage. To address this, you should reference the `Microsoft.Windows.Compatibility` package. The compatibility package includes a broad set of APIs that are common in Windows desktop apps such as WCF client, directory services, registry, configuration, ACLs APIs, and more.
 
-With the Bean Trader sample, the majority of the build errors are due to missing System.ServiceModel types. These could be addressed by referencing the necessary WCF NuGet packages. WCF client APIs are among those present in the [Microsoft.Windows.Compatibility] package, though, so referencing the compatibility package is an even better solution (since it also addresses any issues related to APIs besides the WCF ones that the compatibility package makes available). The [Microsoft.Windows.Compatibility] package helps in most .NET Core 3 WPF and WinForms porting scenarios. After adding the NuGet reference to Microsoft.Windows.Compatibility, only one build error remains!
+With the Bean Trader sample, the majority of the build errors are due to missing System.ServiceModel types. These could be addressed by referencing the necessary WCF NuGet packages. WCF client APIs are among those present in the `Microsoft.Windows.Compatibility` package, though, so referencing the compatibility package is an even better solution (since it also addresses any issues related to APIs besides the WCF ones that the compatibility package makes available). The `Microsoft.Windows.Compatibility` package helps in most .NET Core 3 WPF and WinForms porting scenarios. After adding the NuGet reference to Microsoft.Windows.Compatibility, only one build error remains!
 
 ### Cleaning up unused files
 
 One type of migration issue that may seem obvious but comes up often in practice is issues related to C# and XAML files that weren't previously included in the build getting picked up by the new SDK-style projects that include *all* source automatically.
 
-The next build error we see in the Bean Trader sample refers to a bad interface implementation in OldUnusedViewModel.cs. The file name is a hint, but on inspecting we find that, in fact, this source file is incorrect. It didn't cause issues previously because it wasn't included in the original .NET Framework project. Source files that were present on disk but not included in the old csproj get included automatically now.
+The next build error you see in the Bean Trader sample refers to a bad interface implementation in *OldUnusedViewModel.cs*. The file name is a hint, but on inspecting you find that, in fact, this source file is incorrect. It didn't cause issues previously because it wasn't included in the original .NET Framework project. Source files that were present on disk but not included in the old csproj get included automatically now.
```

Comment:
```suggestion
The next build error you see in the Bean Trader sample refers to a bad interface implementation in *OldUnusedViewModel.cs*. The file name is a hint, but on inspecting you'll find that, in fact, this source file is incorrect. It didn't cause issues previously because it wasn't included in the original .NET Framework project. Source files that were present on disk but not included in the old csproj are included automatically now.
```